firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java

```diff
@@ -53,7 +53,7 @@ class ErrorMessages {
       "The APK failed to install or installation was canceled by the tester.";
 
   static final String API_DISABLED =
-      "The App Distribution Tester API is disabled. It must be enabled in the Google Cloud Console. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.";
+      "The App Distribution Tester API is disabled; you must enable it in the Google Cloud console. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.";
```

Comment:
Could we directly link to the right place in pantheon here? I've seen that in other error messages.

Reply:
Actually we do! We pull it out of the response and append it to the message: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java#L261-L263

You can see an example here: [go/fad-iaf-api-disabled](http://go/fad-iaf-api-disabled)

Reply:
Thanks!